### PR TITLE
feat: Update Components and Nodes UI enhancements

### PR DESCRIPTION
**Pull request by Cristhianzl**
This pull request includes various updates to improve the consistency and clarity of component descriptions and icons across the codebase. The changes primarily focus on updating icon names in different components and JSON files.

### Icon Updates:
* Updated the icon for `GetEnvVar` component from "custom_components" to "code" in `src/backend/base/langflow/components/astra_assistants/getenvvar.py`.
* Updated the icon for `CustomComponent` from "custom_components" to "code" in `src/backend/base/langflow/components/helpers/custom_component.py`.
* Changed the icon for `ChatInput` and `ChatOutput` components from "ChatInput"/"ChatOutput" to "MessagesSquare" in `src/backend/base/langflow/components/inputs/chat.py` and `src/backend/base/langflow/components/outputs/chat.py`. [[1]](diffhunk://#diff-aa0ab7fc66198360550a3153916a174730acf23a984155cc202c13a632952139L12-R12) [[2]](diffhunk://#diff-73b161798dcc5a540208077817f0118793e2edfe20d351e6b66858811b0bf892L11-R11)

### Description Updates:
* Updated the description for `FileComponent` to "Load a file to be used in your project." in `src/backend/base/langflow/components/data/file.py`.

### JSON File Updates:
* Updated the icon for `ChatInput` and `ChatOutput` components from "ChatInput"/"ChatOutput" to "MessagesSquare" in `src/backend/base/langflow/initial_setup/starter_projects/Agent Flow.json` and `Basic Prompting (Hello, World).json`. [[1]](diffhunk://#diff-3c8f8c3614aa5a7d722b91a76b1e62a171db6fd265de720520515475b3e6ef7fL160-R160) [[2]](diffhunk://#diff-3c8f8c3614aa5a7d722b91a76b1e62a171db6fd265de720520515475b3e6ef7fL382-R382) [src/backend/base/langflow/initial_setup/starter_projects/Basic Prompting (Hello, World).jsonL110-R110](diffhunk://#diff-062b4bcff0b8c3b7ea36c54c42f281ecd6896f1d1ca434a20cf727bc21898931L110-R110), [src/backend/base/langflow/initial_setup/starter_projects/Basic Prompting (Hello, World).jsonL447-R447](diffhunk://#diff-062b4bcff0b8c3b7ea36c54c42f281ecd6896f1d1ca434a20cf727bc21898931L447-R447))

### Workflow Update:
* Added a new `npx playwright` command to list tests before running them in `.github/workflows/typescript_test.yml`.**This pull request introduces several key improvements to the visual and functional design of Nodes and Components, aligning with the broader UI Uplift initiative.**

**Summary of Changes**

**Visual Enhancements:**

- The Node component has been resized and adjusted with updated background colors, spacing between elements, and refined icons. Buttons now feature a different hover style, along with tooltips to enhance user experience.

- A basic update to the toolbar has also been implemented to better align with the refreshed Node UI.

**Component Styling:**

- **codeAreaComponent:** Introduced a new icon for accessing the code section, with text animating to give the impression of an action within the component.
- **dictComponent:** Improved button and icon design, and updated hover color to match the new UI palette.
- **dropdownComponent:** Updated component colors to align with the new color scheme.
- **floatComponent:** Added new buttons to increase or decrease field values.
- **inputFileComponent:** Introduced a new file upload icon.
- **inputGlobalComponent:** Added a badge when a value is selected, along with general improvements to color and -layout.
- **inputListComponent:** Standardized design with new icons.
- **promptComponent:** Introduced a new icon for accessing the prompt section, with a visual effect to indicate an action within the component.
- **refreshParameterComponent:** Improved button UI for consistency with other components.
- **strRenderComponent:** Updated the field UI to maintain consistency across components.
- **textAreaComponent:** Introduced a new icon for accessing the text section, with a similar visual action effect.
- **toggleShadComponent:** Adjusted size to be proportional to other elements.

**_These changes aim to create a more cohesive and user-friendly interface, with enhanced visuals and interactions._**

### CODE REVIEW

**Comment by ogabrielluiz** [comment]
hey @Cristhianzl 

Looking good. 

Could you please update the PR title to reflect what this PR does? 

Something like this: https://devguide.python.org/getting-started/pull-request-lifecycle/index.html#making-good-commits

**Review by lucaseduoli — APPROVED** [review]
lgtm!

File: src/frontend/src/CustomNodes/GenericNode/components/NodeDescription/index.tsx [hunk 1]
```diff
@@ -115,7 +114,7 @@ export default function NodeDescription({
           {charLimit && (
             <div
               className={cn(
-                "text-left text-xs",
+                "text-left text-[13px]",
```
Comment (mfortman11) [inline src/frontend/src/CustomNodes/GenericNode/components/NodeDescription/index.tsx:117]: These text elements appear to be 14px in figma

File: src/frontend/src/CustomNodes/GenericNode/components/OutputComponent/index.tsx [hunk 2]
```diff
@@ -24,7 +24,9 @@ export default function OutputComponent({
   };
 
   return displayProxy(
-    <span className={cn(frozen ? "text-ice" : "")}>{name}</span>,
+    <span className={cn("text-[13px] font-medium", frozen ? "text-ice" : "")}>
```
Comment (mfortman11) [inline src/frontend/src/CustomNodes/GenericNode/components/OutputComponent/index.tsx:27]: Id switch this to 12px or 14px it's going to be rare we use an odd number

**Review by mfortman11 — APPROVED** [review]
Looks good just a couple font size comments to keep them to a size that's part of the theme

File: src/frontend/src/CustomNodes/GenericNode/components/NodeDescription/index.tsx (hunk 1 above)
Comment (Cristhianzl) [reply src/frontend/src/CustomNodes/GenericNode/components/NodeDescription/index.tsx:117]: image

thats strange I have 13px on mine...

**Pull request merged**